resources/i18n/en.json

```
@@ -1103,7 +1103,9 @@
     "DarkMode": "Dark Mode",
     "DescThemeMode": "Customize how Backend.AI WebUI looks on your device.",
     "BootstrapScriptEmpty": "The script is empty. \nPlease enter the script.",
-    "BootstrapScriptDeleted": "Bootstrap script deleted"
+    "BootstrapScriptDeleted": "Bootstrap script deleted",
+    "ClassicSessionLauncher": "Classic Session Launcher",
+    "DescClassicSessionLauncher": "Use the classic session launcher instead of the latest session launcher. The classic session launcher may not have the latest features."
```

For this part, can we just call conventional one as 'Classic' and new one as 'Neo'?
for example:
```suggestion
    "DescClassicSessionLauncher": "Use the Classic session launcher instead of Neo. Classic session launcher may not have the latest features."
```